pandas/tests/series/test_timeseries.py

```diff
@@ -370,6 +370,16 @@ def test_pct_change(self, datetime_series):
             rs, (filled / filled.shift(freq="5D") - 1).reindex_like(filled)
         )
 
+    def test_pct_change_with_duplicate_axis(self):
+        # GH 28664
+        common_idx = date_range("2019-11-14", periods=5, freq="D")
+        result = Series(range(5), common_idx).pct_change(freq="B")
```

Comment:
actually can you replicate the original issue here, I think BM was used. I don't mind having B as well.

Reply:
@jreback  I had two options, the original issue used freq BM and periods 70. I thought you also want the explicit and explainable expected, however, if I used periods 70 then the expected would be quite long and hard to explain why it is so.

so two options are as follows:
1. replicate the original issue, but omit the explanation
2. use freq B since it's also anchored offset, and leave the explanation as it is

both options are not hard :D
please tell me which one you more prefer.

Reply:
i c, ok, then let's leave it.